website/docs/r/linux_web_app.html.markdown

```diff
@@ -0,0 +1,555 @@
+
+* `service_tag` - (Optional) The Service Tag used for this IP Restriction.
+
+* `virtual_network_subnet_id` - (Optional) The Virtual Network Subnet ID used for this IP Restriction.
+
+~> **NOTE:** One and only one of `ip_address`, `service_tag` or `virtual_network_subnet_id` must be specified.
+
+---
+
+A `site_config` block supports the following:
+
+* `always_on` - (Optional) If this Linux Web App is Always On enabled. Defaults to `false`.
+
+* `api_management_config_id` - (Optional) The ID of the APIM configuration for this Linux Web App.
+
+* `app_command_line` - (Optional) The App command line to launch.
+
+* `application_stack` - (Optional) A `application_stack` block as defined above.
+
+* `auto_heal` - (Optional) Should Auto heal rules be enabled. Required with `auto_heal_setting`.
+
+* `auto_heal_setting` - (Optional) A `auto_heal_setting` block as defined above. Required with `auto_heal`.
+
+* `auto_swap_slot_name` - (Optional) The Linux Web App Slot Name to automatically swap to when deployment to that slot is successfully completed.
+
+* `cors` - (Optional) A `cors` block as defined above.
+
+* `default_documents` - (Optional) Specifies a list of Default Documents for the Linux Web App.
+
+* `ftps_state` - (Optional) The State of FTP / FTPS service. Possible values include: `AllAllowed`, `FtpsOnly`, `Disabled`.
+
+* `health_check_path` - (Optional) The path to the Health Check.
+
+* `http2_enabled` - (Optional) Should the HTTP2 be enabled?
+
+* `ip_restriction` - (Optional) One or more `ip_restriction` blocks as defined above.
+
+* `load_balancing_mode` - (Optional) The Site load balancing. Possible values include: `WeightedRoundRobin`, `LeastRequests`, `LeastResponseTime`, `WeightedTotalTraffic`, `RequestHash`, `PerSiteRoundRobin`. Defaults to `LeastRequests` if omitted.
+
+* `local_mysql` - (Optional) Use Local MySQL. Defaults to `false`.
+
+* `managed_pipeline_mode` - (Optional) Managed pipeline mode. Possible values include: `Integrated`, `Classic`.
+
+* `minimum_tls_version` - (Optional) The configures the minimum version of TLS required for SSL requests. Possible values include: `1.0`, `1.1`, and  `1.2`.
+
+* `number_of_workers` - (Optional) The number of Workers for this Linux App Service.
+
+* `remote_debugging` - (Optional) Should Remote Debugging be enabled. Defaults to `false`.
+
+* `remote_debugging_version` - (Optional) The Remote Debugging Version. Possible values include `VS2017` and `VS2019`
+
+* `scm_ip_restriction` - (Optional) One or more `scm_ip_restriction` blocks as defined above.
+
+* `scm_minimum_tls_version` - (Optional) The configures the minimum version of TLS required for SSL requests to the SCM site Possible values include: `1.0`, `1.1`, and  `1.2`.
+
+* `scm_use_main_ip_restriction` - (Optional) Should the Linux Web App `ip_restriction` configuration be used for the SCM also.
+
+* `use_32_bit_worker` - (Optional) Should the Linux Web App use a 32-bit worker.
```

Comment:
use seems redundant?
```suggestion
* `32_bit_worker` - (Optional) Should the Linux Web App use a 32-bit worker.
```

Reply:
This was a carry over from the superseded resource, I'll update it 👍